structs.go

```diff
@@ -958,7 +958,7 @@ type GuildTemplate struct {
 	Description string `json:"description"`
```

Comment:
Since we're fixing the `GuildTemplate` struct — this field requires `omitempty` and should be a pointer, to work correctly in [`Modify Guild Template`](https://discord.com/developers/docs/resources/guild-template#modify-guild-template) and it is also optional in [`Create Guild Template`](https://discord.com/developers/docs/resources/guild-template#modify-guild-template). endpoint

```suggestion
	Description *string `json:"description,omitempty"`
```
---

P. S. Same goes for the `name`, to work correctly in [`Modify Guild Template`](https://discord.com/developers/docs/resources/guild-template#modify-guild-template) it should be `omitempty`.